tox.ini

```diff
@@ -33,7 +33,8 @@ ignore =
     FI55,
     FI56,
     FI57,
-    E722
+    E722,
```

Comment:
Please add description for this as a comment, although currently it's not true for other ignores

Reply:
I removed this ignore because I found the flake8 issues could be resolved by removing the requests_mock import statement.  When using it as a pytest fixture, "There is no need to import requests-mock it simply needs to be installed and specify the argument requests_mock."

see https://requests-mock.readthedocs.io/en/latest/pytest.html